examples/calc/src/main.rs

```diff
@@ -3,11 +3,11 @@ use std::io::{self, BufRead, Write};
 #[macro_use] extern crate lrlex;
 
 // Using `lrlex_mod!` brings the lexer for `calc.l` into scope.
-lrlex_mod!(calc);
+lrlex_mod!(calc_l);
```

Comment:
Does the name `calc_l` just come from replacing the periods in the filename with underscores? This might be useful info to add to the comment above (I appreciate, I should have noticed this in a previous PR).

Reply:
Yes, it should be explained in the doc comment for the `process_file_in_src` function. Do you think it needs duplicating here as well?

Reply:
I think it's fair to say that people don't always read docs linearly, or thoroughly! So, yes I think it's probably worth duplicating.

Reply:
^ I think this is the last comment to deal with before squashing.